.gitignore

```diff
@@ -5,3 +5,8 @@ build
 **/node_modules
 **/out
 **/*.tgz
+
+# Visual Studio Related
```

Comment:
we don't really need the comment, it's not like we have 150 things in our gitignore

Reply:
It's still nice to know what entries are associated with which tech, particularly since this repo intends to house various sub projects of different languages.

I can remove it if you feel strongly about it though.

Reply:
i mean `**/bin` is in no way VS specific